test/images/scripts/debian-stable.bootstrap

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh -ex
+ARCH=x86_64
+DEBIAN_LATEST=$(virt-builder -l  | grep "$ARCH" | sort -r | grep -m1 '^debian-' | cut -d' ' -f1)
+exec $(dirname $0)/lib/debian.bootstrap "$1" "$2" "$DEBIAN_LATEST" "deb http://deb.debian.org/debian stable main"
```

Comment:
No, "debian-8" is not a valid release name; that would be "stable", "testing", or "stretch", "jessie", etc. As written in the changelog, the point of this is to take the latest available image for virt-builder and then (possibly) dist-upgrade it to the latest actual stable release. E. g. if Debian 9 "stretch" releases in a few weeks but virt-builder doesn't know about it yet, we'll use debian-8 as a basis and dist-upgrade. The same happens for debian-testing.